packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -205,23 +230,25 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
                 <TypingIndicator {...typingIndicatorProps} styles={typingIndicatorStyles} />
               )}
             </div>
-            <SendBox
-              {...sendBoxProps}
-              autoFocus={options?.autoFocus}
-              styles={sendBoxStyles}
-              /* @conditional-compile-remove(file-sharing) */
-              activeFileUploads={useSelector(fileUploadsSelector).files}
-              /* @conditional-compile-remove(file-sharing) */
-              onCancelFileUpload={adapter.cancelFileUpload}
-            />
-
-            {fileSharing?.uploadHandler && (
-              <FileUploadButton
-                accept={fileSharing?.accept}
-                multiple={fileSharing?.multiple}
-                onChange={fileUploadButtonOnChange}
-              />
-            )}
+            <Stack horizontal={formFactor === 'mobile'} horizontalAlign="start">
+              {formFactor === 'mobile' && (
+                <Stack verticalAlign="center">
+                  <AttachFileButton />
+                </Stack>
+              )}
+              <div style={{ width: '100%' }}>
+                <SendBox
```

Comment:
Are we bringing any of these features to the sendbox component we export?

Reply:
Not 100% sure about the best way to do it.
Since the placement of file attach icon is so different for mobile and desktop.

Reply:
Yeah not sure either, but can always make this decision when we support rich text as that's when many more changes will be made cc @PorterNan

Reply:
Right. That's why I am keeping it simple for now.